lib/checkPullRequestReview.js

```diff
@@ -287,6 +288,75 @@ const handlePullRequestReview = async (context) => {
   }
 };
 
+/**
+ * This function checks when a pull request gets commented on by the PR author.
+ * If pr author comments on the PR asking reviewers please to take a look (ptal),
+ * We check if the reviewers have been assigned and if not, assign them.
+ *
+ * @param {import('probot').Context} context
+ */
+const handleResponseToReview = async (context) => {
+  /**
+   * @type {import('probot').Octokit.IssuesGetResponse}
+   */
+  const pullRequest = context.payload.issue;
+  const isPullRequest = 'pull_request' in pullRequest;
+  if (!isPullRequest) {
+    return;
+  }
+
+  /**
+   * @type {import('probot').Octokit.IssuesGetCommentResponse}
+   */
+  const comment = context.payload.comment;
+  const commenter = comment.user.login;
+  const commenterIsAuthor = commenter === pullRequest.user.login;
+  if (!commenterIsAuthor) {
+    return;
+  }
+
+  const containsPtalComment = PTAL_COMMENTS.some((item) =>
+    comment.body.toLowerCase().includes(item.toLowerCase())
+  );
+  if (!containsPtalComment) {
+    return;
+  }
+
+  // Pause for 3 minutes in case author wants to perform
+  // the required actions.
+  await utilityModule.sleep(THREE_MINUTES);
+
+  // Fetch the pull request in case there has been any changes since
+  // the comment was made.
+  const pullRequestResponse = await context.github.pulls.get(
+    context.repo({
+      pull_number: pullRequest.number,
+    })
+  );
+  const updatedPullRequest = pullRequestResponse.data;
+
+  const usersInComment = utilityModule.getUsernamesFromText(comment.body);
+  // Check if assignment was carried out by author.
+  const assignees = updatedPullRequest.assignees.map((assignee) => assignee.login);
+  const unassignedUsers = usersInComment.filter((user) =>
+    !assignees.includes(user)
+  );
+  console.log({ unassignedUsers });
+  if (unassignedUsers.length === 0) {
+    // All users in comment have been assigned.
+    return;
+  }
+
+  // Assign users in comment
+  context.github.issues.addAssignees(
+    context.repo({
+      issue_number: updatedPullRequest.number,
+      assignees: unassignedUsers,
+    })
+  );
```

Comment:
Add comment too - Hi @ .. assigning you since author requested for a review. Thanks!

Reply:
I'm not sure we should still comment here since we are reacting to a comment already made by the pr author.

Reply:
Hmm, makes sense. But let us unassign the author as well.

Reply:
Done.